lang/nl.po

```diff
@@ -0,0 +1,96 @@
+msgid ""
+msgstr ""
+"Language: nl\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "Dutch translation by Martijn Groeneveldt\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d gratis pakket\n"
+msgstr[1] "%d gratis pakketten\n"
+
+msgid "NONFREE_PACKAGES_COUNT\n"
+msgid_plural "NONFREE_PACKAGES_COUNT\n"
+msgstr[0] "%d niet gratis pakket\n"
+msgstr[1] "%d niet gratis pakketten\n"
+
+msgid "RMS_HAPPY\n"
+msgstr "Alleen gratis pakketten - rms zal trots zijn!\n"
+
+msgid "RMS_DISAPPOINTED\n"
+msgstr "Meer dan 10%% propriëtaire pakketten. Heeft u geen interesse in vrijheid?\n"
+
+msgid "HELP_USAGE\n"
+msgstr "gebruik: vrms-rpm [opties]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Laat rms ASCII-kunst zien wanneer er geen propriëtaire softwarepakketten gevonden zijn\n"
```

Comment:
This line should wrap at 80 characters. You can split the message into fewer or more lines, as needed.